src/porepy/fracs/meshing.py

```diff
@@ -240,7 +240,7 @@ def _tag_faces(grids, check_highest_dim=True):
 
     # Assume only one grid of highest dimension
     if check_highest_dim:
-        assert len(grids[0]) == 1, "Must be exactly" "1 grid of dim: " + str(len(grids))
+        assert len(grids[0]) == 1, "Must be exactly1 grid of dim: " + str(len(grids))
```

Comment:
```suggestion
        assert len(grids[0]) == 1, "There must be exactly 1 grid of dim: " + str(len(grids))
```

Reply:
Note: I have a feeling this might have been introduced by `ruff`'s `--fix` option. This may be something to be aware of.